### PR TITLE
[teleop] ERD and ES gui redesign

### PR DESCRIPTION
**Pull request by catkims**
Menu:
image

ERD GUI:
image
image
image
image

ES GUI:
image
image
image

### CODE REVIEW

File: base_station/gui/src/components/EncoderCounts.vue [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-    <div>joint f: {{Math.round((encoderCounts.joint_f + Number.EPSILON) * 10000) / 10000}}</div>
+  <div class="wrap">
+    <h3>Encoder Counts</h3>
+    <div class="encoders">
+      <div>joint a: {{Math.round((encoderCounts.joint_a + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint b: {{Math.round((encoderCounts.joint_b + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint c: {{Math.round((encoderCounts.joint_c + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint d: {{Math.round((encoderCounts.joint_d + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint e: {{Math.round((encoderCounts.joint_e + Number.EPSILON) * 10000) / 10000}}</div>
+      <div>joint f: {{Math.round((encoderCounts.joint_f + Number.EPSILON) * 10000) / 10000}}</div>
```
Comment (CameronTressler) [inline base_station/gui/src/components/EncoderCounts.vue:10]: Can you fix this rounding? I think I implemented this rounding really quickly and didn't realize I could just use .toFixed(5)

File: base_station/gui/src/components/GimbalControls.vue [hunk 2]
```diff
@@ -55,4 +55,10 @@ export default {
...
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+  .wrap {
+    display: inline-block;
+  }
+</style>
```
Comment (CameronTressler) [inline base_station/gui/src/components/GimbalControls.vue:64]: Nitpick: add a newline

File: base_station/gui/src/components/PDBFuse.vue [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+      if (val > threshold || val < 0.0) {
+        struct.color = "red"
+      }
+      else {
+        struct.color = "black"
+      }
+    }
+  }
+}
+</script>
```
Comment (CameronTressler) [inline base_station/gui/src/components/PDBFuse.vue:290]: nitpick: add a newline

**Comment by tabiosg** [comment]
Resolve conflicts

**Review by tabiosg — CHANGES_REQUESTED** [review]
merge conflicts

**Review by CameronTressler — APPROVED** [review]

**Review by tabiosg — APPROVED** [review]

**Pull request merged**